setup.py

```diff
@@ -30,25 +31,28 @@ def local_open(fname):
 for directory in os.walk('example/templates'):
     example_dirs.append(directory[0][8:] + '/*.*')
 
+with local_open(os.path.join('pages', '__init__.py')) as f:
+    exec(f.read(), {}, pages)
```

Comment:
It was working because your `__init__.py` is empty (no `import`).
It's failling because `requirements-builder` does an `eval` on `setup.py` without that specific folder into the `sys.path`.

Maybe the problem lies in `requirements-builder` and it should hack the `sys.path` too.

@jirikuncar any thoughts?